projects/igniteui-angular/src/lib/tabs/tabs/tabs.component.ts

```diff
@@ -119,8 +120,8 @@ export class IgxTabsComponent extends IgxTabsDirective implements AfterViewInit,
     private _tabAlignment: string | IgxTabsAlignment = 'start';
     private _resizeObserver: ResizeObserver;
 
-    constructor(builder: AnimationBuilder, cdr: ChangeDetectorRef, private ngZone: NgZone, public dir: IgxDirectionality) {
-        super(builder, cdr, dir);
+    constructor(@Inject(IgxAngularAnimationService) animationService: AnimationService, cdr: ChangeDetectorRef, private ngZone: NgZone, public dir: IgxDirectionality) {
```

Comment:
```suggestion
    constructor(@Inject(IgxAngularAnimationService) private animationService: AnimationService, cdr: ChangeDetectorRef, private ngZone: NgZone, public dir: IgxDirectionality) {
```

Reply:
this is already marked as private in base class